### PR TITLE
TDG-56-user-story-17-list-files-by-templates-and-filled-documents

### PR DESCRIPTION
**Pull request by leand12**

### CODE REVIEW

File: RetrieveService/src/main.py [hunk 1]
```diff
@@ -29,8 +29,13 @@ async def get_templates(s3=Depends(get_client_s3)) -> JSONResponse:
             Json response with the status code and data containing
             the message and data.
     """
+    prefix = ''
+    if file_type in ('filled', 'template'):
+        prefix = file_type + '/'
     try:
-        file_contents = s3.list_objects_v2(Bucket=bucket_name)["Contents"]
+        print(prefix)
```
Comment (BrunosBastos) [inline RetrieveService/src/main.py]: Remove this

File: RetrieveService/src/main.py [hunk 2]
```diff
@@ -29,8 +29,13 @@ async def get_templates(s3=Depends(get_client_s3)) -> JSONResponse:
             Json response with the status code and data containing
             the message and data.
     """
+    prefix = ''
+    if file_type in ('filled', 'template'):
```
Comment (BrunosBastos) [inline RetrieveService/src/main.py]: If the file type is not supported then it should return a response with an error.

Comment (leand12) [reply RetrieveService/src/main.py]: done

File: RetrieveService/src/main.py (hunk 1 above)
Comment (leand12) [reply RetrieveService/src/main.py]: done

**Pull request merged**